src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -1,3 +1,4 @@
+using Agent.Sdk;
```

Comment:
This file is going to have conflicts with changes I am working on. You may want to revert this one as well.